src/Sulu/Bundle/MediaBundle/Media/Manager/MediaManager.php

```diff
@@ -212,10 +212,16 @@ public function getEntityById($id)
         return $mediaEntity;
     }
 
-    public function getByIds(array $ids, $locale)
+    public function getByIds(array $ids, $locale, UserInterface $user = null, $permission = null)
```

Comment:
I just hope that nobody relies on the current behavior...

Reply:
Old Behaviour. Did I oversee something? There was no user before in the used interfaces or? Then I would keep it as it is here currently.

Reply:
No, it was not, but it also `null` differently now. People might expect to get all the entities anyway, but if there is an anonymous role with some missing permissions, this function might not return all the passed ids anymore.

Reply:
Okay I understand no then we should definitely send the $permission else it will accidently filter out media when using the command from the CLI context.

Reply:
I think I would keep it without the `permission` variable for now... Why should somebody use this method to get all the media? There are other methods for that... And as long as they are not introducing any permissions on the security it will behave the same, so they probably cannot even tell if that is a BC break :see_no_evil: